.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+	  "editor.tabSize": 2,
+    "editor.detectIndentation": false,
+}
```

Comment:
I can - and will - exclude this commit, I'm using vscode and it started messing with the indentation.